### PR TITLE
Adding debug-socket bin script

### PR DESCRIPTION
**Pull request by mdayaram**
Added a helper script to interact with the created debug-sockets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ end
 Then in a terminal:
 
 ```
-% echo backtrace | socat - UNIX-CONNECT:~/tmp/puma-debug-1234.sock
+% debug-socket ~/tmp/puma-debug-1234.sock backtrace
```
Comment (nerdrew) [inline README.md]: Should it default to `backtrace`?

**Comment by nerdrew** [comment]
Thanks @mdayaram!

File: README.md (hunk 1 above)
Comment (mdayaram) [inline README.md]: True!  Done!

**Comment by mdayaram** [comment]
Okay, tested and works fine, also updated the README.

**Pull request merged**